pipelines/live-1/main/tzcronjobber.yaml

```diff
@@ -0,0 +1,73 @@
+resources:
+- name: tools-image
+  type: docker-image
+  source:
+    repository: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tools
+    tag: latest
+    aws_access_key_id: ((aws-live-1.access-key-id))
+    aws_secret_access_key: ((aws-live-1.secret-access-key))
+- name: cronjobber-image
+  type: docker-image
+  source:
+    repository: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tools
+    tag: cronjobber
+    aws_access_key_id: ((aws-live-1.access-key-id))
+    aws_secret_access_key: ((aws-live-1.secret-access-key))
+jobs:
+  - name: cronjobber
+    plan:
+      - get: tools-image
+      - task: download
+        image: tools-image
+        config:
+          platform: linux
+          params:
+            RELEASE: 0.2.0
+          run:
+            path: /bin/bash
+            dir: files
+            args:
+              - -exc
+              - |
+                wget -q https://github.com/hiddeco/cronjobber/archive/${RELEASE}.tar.gz
+                tar xzf ${RELEASE}.tar.gz --strip-components=1
+          outputs:
+            - name: files
+      - put: cronjobber-image
+        inputs: [files]
+        params:
+          build: files
+          dockerfile: files/Dockerfile
+          tag_as_latest: false
+          cache: true
+      - task: install
+        image: tools-image
+        config:
+          platform: linux
+          params:
+            AWS_ACCESS_KEY_ID: ((aws-live-1.access-key-id))
+            AWS_SECRET_ACCESS_KEY: ((aws-live-1.secret-access-key))
+            KUBECONFIG: /tmp/kubeconfig
+            ECR: 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/tools
+            RELEASE: 0.2.0
+          inputs:
+            - name: files
+          run:
+            path: /bin/bash
+            args:
+              - -exc
+              - |
+                apk add sed
+                pushd files
+                cp deploy/crd.yaml ../cronjobber-crd.yaml
+                cp deploy/deploy.yaml ../cronjobber-deploy.yaml
+                cp deploy/rbac.yaml ../cronjobber-rbac.yaml
+                popd
+                sed -i 's/^metadata:/metadata:\n  namespace: kube-system/g' cronjobber-crd.yaml
```

Comment:
Instead of copying these files and injecting the namespace with `sed`, we can simply specify when applying: `kubectl -n kube-system apply -f crd.yaml -f deploy.yaml -f rbac.yaml`. Any reason why this wouldn't work?

I think it would make this script a lot simpler and easier to understand.